pyresample/test/test_geometry.py

```diff
@@ -2037,6 +2037,32 @@ def test_get_lonlats(self):
         np.testing.assert_allclose(lats[:464, :], lats0)
         np.testing.assert_allclose(lats[464:, :], lats1)
 
+        # check get_lonlats with chunks definition doesn't cause errors and arrays are equal
+        # too many chunks
+        lons_c, lats_c = final_area.get_lonlats(chunks=((2000, 3568, 5568), (464,)))
+        np.testing.assert_array_equal(lons, lons_c)
+        np.testing.assert_array_equal(lats, lats_c)
```

Comment:
is it really array equality you want to check, or rather that `lons_c` and `lats_c` have the right chunk sizes?

Reply:
Good question. My thinking was: if the arrays are equal, the reading - and therefore the chunking - worked out (didn't fail) and we're good. So I was rather testing the final result rather than the actual implementation. Which is the best way to go?

Reply:
So you are actually testing that the `get_lonlats` call doesn't crash right? So it is the computation you are after then, and the array equality assert triggers that. So I think we're good :)

Reply:
Yes, thanks!